website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -86,6 +86,19 @@ metrics:
 | filters     | A list of filters to apply before calculating the metric    | See below                       | no        |
 | meta        | Arbitrary key/value store                                   | {team: Finance}                 | no        |
 
+### Filters
+Filters should be defined as a list of dictionaries that define predicates for the metric. Filters are ANDed together. If more complex filtering is required, users can (and should) push that logic down into the underlying model.
```

Comment:
Hi @joellabes just some fly by questions here! 

> Filters are ANDed together
Does this mean that when you provide multiple filters they will use AND logic, meaning all criteria must be met? It might help newer users and users who aren't native English speakers to expand this sentence just a bit! (Although I do appreciate how efficient this sentence is. ❤️ )

> ...pushing that logic down into the underlying model.
I'm also curious about this phrase. Do we talk more anywhere on how to do this? Can we link people to this?

Reply:
> Does this mean that when you provide multiple filters they will use AND logic

Yep! happy to have it changed to anything else - none of this is my writing so I'm even less precious about it than normal 😉 

>I'm also curious about this phrase
Basically, instead of trying to implement more complex filters in the metric definition, we're expecting people to do their complex filtering inside of the `model` (in this example, `ref('dim_customers')`). I don't know if there's prior art we can link people out to :(